update-all

```diff
@@ -12,6 +12,12 @@ branches=(
     3.3-testing
     3.3-testing-el6
     3.3-testing-el7
+    3.4-devel
+    3.4-devel-el6
+    3.4-devel-el7
+    3.4-testing
+    3.4-testing-el6
+    3.4-testing-el7
```

Comment:
Those were actually in my `stash`  :)
[](url)
I think at first they were [omitted intentionally](https://github.com/opensciencegrid/docker-osg-wn-scripts/pull/1/#issuecomment-308776219) and were going to add back in the prod branches in a separate PR.  But anyway, they're added back now.